apps/site/lib/site_web/controllers/schedule/line_api.ex

```diff
@@ -34,12 +34,7 @@ defmodule SiteWeb.ScheduleController.LineApi do
           |> assign(:direction_id, String.to_integer(direction_id))
           |> assign_alerts(filter_by_direction?: true)
 
-        line_data =
-          get_line_data(
-            route,
-            String.to_integer(direction_id),
-            conn.query_params["route_pattern"]
-          )
+        line_data = get_line_data(route, String.to_integer(direction_id))
```

Comment:
I _suspect_ it was the removal of `conn.query_params["route_pattern"]` here that caused this regression - at left is this PR - it's no longer displaying the extra stops for Inbound - Haymarket via Forest Hills. Variants on buses, oof.

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/2136286/185693334-31dd4a7d-daad-47ef-91d4-baccd1a0cfb8.png">

It's maybe easiest to see on the 62/76 - https://dev-green.mbtace.com/schedules/627/line?schedule_direction%5Bdirection_id%5D=0&schedule_direction%5Bvariant%5D=627-7-0 One expects after selecting the "via Hanscom" variant to be able to see the Hanscom stop!